functional/FunctionalTest.java

```diff
@@ -4112,10 +4187,7 @@ public static void runTests() throws Exception {
     selectObjectContent_test1();
 
     // SSE_C tests will only work over TLS connection
-    Locale locale = Locale.ENGLISH;
-    boolean tlsEnabled = endpoint.toLowerCase(locale).contains("https://");
-    if (tlsEnabled) {
-      statObject_test2();
+    if (endpoint.toLowerCase(Locale.US).contains("https://")) {
```

Comment:
I see the same condition for `statObject_test2()` as well. Maybe there can be a single `if` block for all the tests that depend on this condition?